### PR TITLE
replaced: invalid vmSizes for Ubuntu 14.04.5-LTS

### PR DESCRIPTION
**Pull request by marlonsingleton**
### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use a parameter for resource locations with the defaultValue set to resourceGroup().location
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values)
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md

- [X] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

* added valid vmSizes
* removed failing special place holders
* updated parameters to generate unique strings

### Reason

* current syntax causes deployment to fail

### CODE REVIEW

**Comment by marlonsingleton** [comment]
@bmoore-msft please review. Thanks!

File: ubuntu-desktop-gnome/azuredeploy.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "parameters": {
     "newStorageAccountName": {
       "type": "string",
+      "defaultValue": "[concat(toLower(uniqueString(resourceGroup().id)), 'storage')]",
```
Comment (bmoore-msft) [inline ubuntu-desktop-gnome/azuredeploy.json]: revert this change

File: ubuntu-desktop-gnome/azuredeploy.json [hunk 2]
```diff
@@ -17,28 +18,31 @@
     },
     "adminPassword": {
       "type": "securestring",
+      "defaultValue": "[base64(concat(substring(uniqueString(deployment().name), 3, 9), uniqueString(resourceGroup().id)))]",
```
Comment (bmoore-msft) [inline ubuntu-desktop-gnome/azuredeploy.json]: remove the defaultValue on the password

File: ubuntu-desktop-gnome/azuredeploy.json [hunk 3]
```diff
@@ -17,28 +18,31 @@
...
       "metadata": {
-        "description": "Password for the Virtual Machine."
+        "description": "Password for the Virtual Machine. Use a deployment name to increase complexity."
       }
     },
     "dnsNameForPublicIP": {
       "type": "string",
+      "defaultValue": "[concat('dns', substring(parameters('newStorageAccountName'), 0, 6))]",
```
Comment (bmoore-msft) [inline ubuntu-desktop-gnome/azuredeploy.json]: revert this change

File: ubuntu-desktop-gnome/azuredeploy.parameters.json [hunk 4]
```diff
@@ -1,19 +1,5 @@
...
-            "value": "GEN-UNIQUE-8"
-        },
-        "adminPassword": {
-            "value": "GEN-PASSWORD"
-        },
-        "dnsNameForPublicIP": {
-            "value": "GEN-UNIQUE"
-        }
-    }
+    "parameters": { }
```
Comment (bmoore-msft) [inline ubuntu-desktop-gnome/azuredeploy.parameters.json]: revert this change

Comment (marlonsingleton) [reply ubuntu-desktop-gnome/azuredeploy.parameters.json]: reverted changes, CI test passed. Thanks!

**Comment by marlonsingleton** [comment]
Merge request a bit messy. I'll submit a new PR.
